bin/webpack/createConfig.js

```diff
@@ -2,7 +2,9 @@ function createConfig(options) {
   const path = require("path");
   const webpack = require("webpack");
   const WebpackNotifierPlugin = require("webpack-notifier");
+  var HappyPack = require('happypack');
```

Comment:
Nice! Happypack definitely gives a solid speed boost. I've had some issues in the past with non-deterministic builds (File changes despite no actual changes,) but out build is simple enough that I don't think this'll be an issue.